packages/fuselage/src/components/Option/Option.tsx

```diff
@@ -49,6 +49,7 @@ const Option = memo(
       id={id}
       ref={ref}
       aria-selected={selected}
+      aria-disabled={disabled}
```

Comment:
I guess it should be an string ("true" or "false"):

```suggestion
      aria-disabled={String(disabled)}
```